Remove casual language describing our products

paulkel opened the pull request:
Replaced the word "flavors" that was used to describe versions of SQL Server with "versions". We should be clear in our language, using terms such as "versions" and "editions" that we officially use to describe variations in our products rather than casual and ambiguous terms like "flavor".

azurecla commented:
Hi __@ShaneCastle__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Jim-Parker commented:
at @ShaneCastle Shane, thank you for making the updates to the Service Bus brokered messaging .NET tutorial.

Jim-Parker commented:
@sethmanheim Seth, could you please review this contribution?

sethmanheim commented:
@ShaneCastle Thanks for catching that, we lost some spaces when we converted from MSDN to Azure.com. :-) We'll get your contribution published right away. #sign-off

paulkel committed:
Remove casual language describing our products

Replaced the word "flavors" that was used to describe versions of SQL Server with "versions". We should be clear in our language, using terms such as "versions" and "editions" that we officially use to describe variations in our products rather than casual and ambiguous terms like "flavor".
